### ISSUE TITLE
[stable/mariadb] 'Access denied for user 'root'@'localhost' (using password: YES)'

### ISSUE THREAD

**Issue by 4ydx** [helm/charts#9524]
**BUG REPORT

**Version of Helm and Kubernetes**:
minikube version: v0.30.0
Helm:
Client: &version.Version{SemVer:"v2.11.0", GitCommit:"2e55dbe1fdb5fdb96b75ff144a339489417b146b", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.11.0", GitCommit:"2e55dbe1fdb5fdb96b75ff144a339489417b146b", GitTreeState:"clean"}

**Which chart**:
stable/mariadb

**What happened**:
Seeing the following error:
Readiness probe failed:  mysqladmin: connect to server at 'localhost' failed error: 'Access denied for user 'root'@'localhost' (using password: YES)' 

**What you expected to happen**:minikube version: v0.30.0
That this not occur.

**How to reproduce it** (as minimally and precisely as possible):
1) Fresh clone of helm/charts.
2) helm install --name mariadb-service -f values.yaml . --tls > OUTPUT
3) Observe the minikube dashboard --url

When logged into the master pod, the auto-generated password does not work either.  I assume the same must be true for the slave.

**Comment by 4ydx** [comment]
I guess I will have to look into this further.  

I just created an entirely new cluster in minikube and the first thing I did was start up stable/mariadb.  Just as above I didn't alter anything about the mariadb settings, but this time the mysqladmin check is passing.

Previously I had spun up (install followed by delete --purge) stable/mariadb with different settings a few times before reverting to the blank slate.  The reason for the install/delete cycle is that the mysqladmin check immediately failed from the beginning for me.  I don't know how this might have caused a difference.  I will see if I can sort out precisely what it was that I did to get this to occur, but if anyone has ideas I would appreciate it.
